client/app/stores/requests_in_flight_store.coffee

```diff
@@ -0,0 +1,35 @@
+Immutable = require 'immutable'
```

Comment:
This is need to keep tracks of all requests performed in background. We choose to use it previously with @aenario when designing the new architecture. It's also a needed component for a stateless version that'll comes next. For all of this, I've choose to introduce it now. Do it appeal to you, or should we move to another pattern?